docs/content/build-and-deploy.md

```diff
@@ -60,10 +60,11 @@ Use the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/i
 
 ### Cloud deployment
 To start a cloud deployment, follow these steps: 
-1. Upload your server-worker and client-worker assemblies. To do this, in a terminal window, from any directory, run `spatial cloud upload <assembly-name>` from the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli). 
+1. Specify the SpatialOS project name you want to deploy to. To do this, locate the spatialos.json file in the root folder of your SpatialOS project and edit the file so the name field matches your SpatialOS project name. You can find your project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something like `beta_someword_anotherword_000`. (To set up your project in SpatialOS, follow the instructions in the [Setup guide](../setup-and-installing.md).)
+2. Upload your server-worker and client-worker assemblies. To do this, in a terminal window, from any directory in your project, run `spatial cloud upload <assembly-name>` from the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli).
 </br>You can choose any assembly name you want (but note that the name can only contain alpha-numerical characters, hyphens, dots and underscores, and must be between 5 and 64 characters long).
-2. Once the server-worker and client-worker assemblies are successfully uploaded, from the same directory run `spatial cloud launch <assembly-name> <path/to/launch-config.json> <deployment-name> --snapshot=<path/to/snapshot.snapshot>`, replacing the `<example-content>` with file path and file names relevant to your game. For more information on snapshots, see [Snapshots](snapshots.md).
-3. The `spatial cloud launch` command automatically opens the SpatialOS [Console](https://docs.improbable.io/reference/latest/shared/glossary#console) in your browser window once the deployment has started. Note that this process might take a couple of minutes. You can then view what’s happening in your game’s deployment in real time.
+3. Once the server-worker and client-worker assemblies are successfully uploaded, from the same directory run `spatial cloud launch <assembly-name> <path/to/launch-config.json> <deployment-name> --snapshot=<path/to/snapshot.snapshot>`, replacing the `<example-content>` with file path and file names relevant to your game. For more information on snapshots, see [Snapshots](snapshots.md).
+4. The `spatial cloud launch` command automatically opens the SpatialOS [Console](https://docs.improbable.io/reference/latest/shared/glossary#console) in your browser window once the deployment has started. Note that this process might take a couple of minutes. You can then view what’s happening in your game’s deployment in real time.
```

Comment:
There's a whole section in @jamiebrynes7 installer docs detailing the spatial cloud launch arguments, which I think should be in here. 
It is below:

---

 - **About the `spatial cloud launch` command**
<br/> The command `spatial cloud launch` deploys a project to the cloud. Its syntax is:
        `spatial cloud launch <assembly name> <launch configuration> <deployment name> --snapshot=<path/to/snapshot.snapshot>`
        where:
        - `<assembly name>` is the name of the assembly the deployment will use (the one you named).
        - `<launch configuration>` is the configuration file for the deployment. The GDK expects to find this in <WHERE???>.
        - `<deployment name>` you choose as you enter the command; you’ll use this to identify the deployment. This must be in lowercase.
        - `<snapshot file>` is the snapshot of the world you want to start from.  See the the GDK documentation for further information on [snapshots](./content/snapshots.md).

---